moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -172,10 +172,22 @@ export const useColumns = ({ hiddenColumns }) => {
             state={{ queryString }}
             sx={{ color: theme.palette.primary.main }}
           >
-            {row.project_name}
+            {row.project_name_full}
           </Link>
         ),
       },
+      {
+        headerName: "Name",
+        field: "project_name",
+        flex: 2,
+        minWidth: COLUMN_WIDTHS.xlarge,
+      },
+      {
+        headerName: "Secondary name",
+        field: "project_name_secondary",
+        flex: 2,
+        minWidth: COLUMN_WIDTHS.xlarge,
```

Comment:
this feels too wide to me, but then again I don't know how long project secondary names will be....